cmd/deviceservice/update/update.go

```diff
@@ -0,0 +1,149 @@
+package update
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"html/template"
+	"io/ioutil"
+
+	"github.com/edgexfoundry-holding/edgex-cli/config"
+	"github.com/edgexfoundry-holding/edgex-cli/pkg/editor"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/metadata"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/urlclient/local"
+	"github.com/edgexfoundry/go-mod-core-contracts/models"
+
+	"github.com/spf13/cobra"
+)
+
+const DeviceSrviceTempl = `{
+   "Id": "{{.Id}}",
+   "Name" : "{{.Name}}",
+   "Description" : "{{.Description}}",
+   "AdminState" : "{{.AdminState}}",
+   "OperatingState" : "{{.OperatingState}}",
+   "Labels" : [
+    {{- $labelsLenght := len .Labels}}
+    {{- range $idx, $l := .Labels}}
+      "{{$l}}" {{if not (lastElem $idx $labelsLenght)}},{{end}} 
+    {{- end}}
+    ],
+   "Addressable" :
+   {
+     "Name" : "{{.Addressable.Name}}"
+   }
+ }`
+
+var name string
+var file string
+
+// NewCommand returns the update device service command
+func NewCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update",
+		Short: "Update device service",
+		Long:  `Update device service(s) described in the given JSON file or use the interactive mode with additional flags.`,
+		RunE:  deviceServiceHandler,
+	}
+	cmd.Flags().StringVarP(&name, "name", "n", "", "Name")
+	cmd.Flags().StringVarP(&file, "file", "f", "", "Json file containing device service configuration to update")
+	return cmd
+}
+
+func deviceServiceHandler(cmd *cobra.Command, args []string) error {
+	if name != "" && file != "" {
+		return errors.New("DeviceService could be updated by providing a file, or by specifying device service name to be updated and using interactive mode. ")
```

Comment:
s/DeviceService/DeviceService(s)

Reply:
This comment is already addressed in PR #271
The current PR depends on #271 and this is why the change from both PRs are visible here.
Better approach probably was to wait the first PR to be merged.